packages/utxo-lib/src/compose/index.ts

```diff
@@ -47,14 +47,14 @@ export function composeTx({
 
     const splitOutputs = request.splitByCompleteness(outputs);
 
-    let csResult: Result = { type: 'false' };
+    let csResult: ReturnType<typeof coinselect> = { type: 'false' };
```

Comment:
Why this change? Also, `type Result` in `coinselect` is now unused

Reply:
because it's cleaner :) and type is still used (componse/coinselect line 44)

Reply:
True but the `export` is meaningless. That's a nitpick though

Reply:
feel free to remove the export then